samples/snippets/quickstart_create_saved_query_test.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import uuid
+
+import quickstart_create_saved_query
+
+
+PROJECT = os.environ["GOOGLE_CLOUD_PROJECT"]
+SAVED_QUERY_ID = "saved-query-{}".format(uuid.uuid4().hex)
+
+
+def test_create_saved_query(capsys, saved_query_deleter):
+    saved_query = quickstart_create_saved_query.create_saved_query(
+        PROJECT, SAVED_QUERY_ID, "saved query foo")
+    saved_query_deleter.append(saved_query.name)
+    out, _ = capsys.readouterr()
+    assert "saved_query" in out
```

Comment:
All the tests in general only seem to be checking that RPC calls are made. The tests should check that (1) RPC calls are made, but also (2) check the contents of the response (delete doesn't return the response in the API so listing for empty queries or something would also work), to ensure that the tests are accurate.

Reply:
Closing the loop: not in scope for this sample. Current checks are fine.